nvflare/app_opt/lightning/fl_callbacks.py

```diff
@@ -0,0 +1,99 @@
+from enum import IntEnum
+
+import pytorch_lightning as pl
+from pytorch_lightning.trainer.states import TrainerFn
+
+import nvflare.client as flare
+from nvflare.app_common.abstract.fl_model import FLModel
+from nvflare.client.config import ConfigKey
+from nvflare.client.constants import ModelExchangeFormat
+
+
+class SendTrigger(IntEnum):
```

Comment:
Not sure if Python 3.8 has IntEnum.

That leads to a question:
Are we saying we only support Python 3.9+? if so we are good with this code.
IF not, we need to change this.